Vizzuality/half-earth-v3
# Fix/layer ordering bug

## Fix layer ordering after reload (or share)
### Description
With the latest changes on the ordering function (`setLayersOrder`) the basemap layer (firefly or vibrant) was being rendered on top of some of the data layers.
In order to avoid this we have defined a function to set the basemap of the different maps. Since we are setting the basemaps on the code, the [WebScene](https://developers.arcgis.com/javascript/latest/api-reference/esri-WebScene.html) that was being used as the base for the [SceneView](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-SceneView.html) has been replaced with a [Map](https://developers.arcgis.com/javascript/latest/api-reference/esri-Map.html). This way the semantics of the props are more accurate and we avoid dealing with no needed scene ids. This webScene bit was a remanent from the times we were storing all scene layers on a webScene on the arcgis online account, since this is not happening anymore we can get rid of it and use a Map instead.
### Designs
_No designs needed_
### Testing instructions
This is the link shared by the user with the bug happening [bit.ly/38GPXuG](bit.ly/38GPXuG). To reproduce the actual bug on [the production site](https://half-earth.vizzuality.com/dataGlobe) you need to turn on all the human pressures layers (including the marine one) and then copy paste the url in a new tab. You'll see that some layers are hidden behind the basemap (this happens because the basemap is not a propper basemap but a layer and we where trying to position the layer when setting the layers order).
Make sure you check all the scenes in the app are working as expected.
### Feature relevant tickets
This bug was reported from a user.
See this [slack thread](https://vizzuality.slack.com/archives/CCW79P5C1/p1605248190022400) for reference.
